e2e-tests/gef/cypress/e2e/facility/pre-submission/facility-end-date-enabled/facilities-change-facility-end-date-to-bank-review-date-journey.spec.js

```diff
@@ -0,0 +1,112 @@
+    cy.keyboardInput(aboutFacility.coverEndDateDay(), todayDay);
+    cy.keyboardInput(aboutFacility.coverEndDateMonth(), todayMonth);
+    cy.keyboardInput(aboutFacility.coverEndDateYear(), nextYear);
+    aboutFacility.isUsingFacilityEndDateNo().click();
+    cy.clickContinueButton();
+
+    cy.url().should('eq', relative(`/gef/application-details/${application.id}/facilities/${facilityId}/bank-review-date`));
+    cy.fillInBankReviewDate(tomorrow);
+    cy.clickContinueButton();
+
+    cy.url().should('eq', relative(`/gef/application-details/${application.id}/facilities/${facilityId}/provided-facility`));
+    cy.clickBackLink();
+
+    cy.url().should('eq', relative(`/gef/application-details/${application.id}/facilities/${facilityId}/bank-review-date`));
+    cy.clickBackLink();
+
+    cy.url().should('eq', relative(`/gef/application-details/${application.id}/facilities/${facilityId}/about-facility`));
+    aboutFacility.isUsingFacilityEndDateYes().click();
+    cy.clickContinueButton();
+
+    cy.url().should('eq', relative(`/gef/application-details/${application.id}/facilities/${facilityId}/facility-end-date`));
+    cy.keyboardInput(facilityEndDate.facilityEndDateDay(), tomorrowDay);
+    cy.keyboardInput(facilityEndDate.facilityEndDateMonth(), tomorrowMonth);
+    cy.keyboardInput(facilityEndDate.facilityEndDateYear(), tomorrowYear);
+    cy.clickContinueButton();
+
+    cy.url().should('eq', relative(`/gef/application-details/${application.id}/facilities/${facilityId}/provided-facility`));
+    cy.clickBackLink();
+
+    cy.url().should('eq', relative(`/gef/application-details/${application.id}/facilities/${facilityId}/facility-end-date`));
+    cy.clickBackLink();
+
+    cy.url().should('eq', relative(`/gef/application-details/${application.id}/facilities/${facilityId}/about-facility`));
+    aboutFacility.isUsingFacilityEndDateNo().click();
+    cy.clickContinueButton();
+
+    cy.url().should('eq', relative(`/gef/application-details/${application.id}/facilities/${facilityId}/bank-review-date`));
+    bankReviewDate.bankReviewDateDay().should('have.value', '');
+    bankReviewDate.bankReviewDateMonth().should('have.value', '');
+    bankReviewDate.bankReviewDateYear().should('have.value', '');
+
+    cy.fillInBankReviewDate(tomorrow);
+    cy.clickContinueButton();
+
+    cy.url().should('eq', relative(`/gef/application-details/${application.id}/facilities/${facilityId}/provided-facility`));
+    cy.clickBackLink();
+
+    cy.url().should('eq', relative(`/gef/application-details/${application.id}/facilities/${facilityId}/bank-review-date`));
+    cy.clickBackLink();
+
+    cy.url().should('eq', relative(`/gef/application-details/${application.id}/facilities/${facilityId}/about-facility`));
+    aboutFacility.isUsingFacilityEndDateYes().click();
+    cy.clickContinueButton();
+
+    cy.url().should('eq', relative(`/gef/application-details/${application.id}/facilities/${facilityId}/facility-end-date`));
```

Comment:
Lots of repeated URLs in `cy.visit` and URL assertions - would be nice to create some DRY consts e.g `facilityEndDateUrl`, `aboutFacilityUrl`

This would be a new pattern though, not sure if we should do this, would be better to do in one go, which'd be epic, otherwise tests would be misaligned